### DISCUSSION TITLE
Can I ask my friend to pay for Github Copilot from a different country?

### DISCUSSION THREAD

**Discussion by fermanentdev** [community/community#128542] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, If I ask my friend to pay for Github Copilot from a different country will i get ban?
(i need VPN for copilot :D)

**Comment by Pinkolinka11** [comment]
****wdz

**Comment by Pinkolinka11** [comment]
> 

> wdgk

**Comment by Pinkolinka11** [comment]
> 

> __

**Comment by mst-rajatmishra** [comment]
NO I GUESS, BUT VPN CAN MAKE THAT BAN!,
